### PR TITLE
Adding customized_wait

### PR DESCRIPTION
**Pull request by VinithaDar**

### CODE REVIEW

File: SeleniumPactice/April2022SeleniumSession/src/main/java/seleniumsessions/CustomizedWait.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package seleniumsessions;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
```
Comment (snelloru) [inline SeleniumPactice/April2022SeleniumSession/src/main/java/seleniumsessions/CustomizedWait.java:6]: @VinithaDar Do we need to import this here?

File: SeleniumPactice/April2022SeleniumSession/src/main/java/seleniumsessions/CustomizedWait.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	public static void main(String[] args) {
+		
+		WebDriverManager.chromedriver().setup();
+		driver = new ChromeDriver();
+		
+		driver.get("https://www.amazon.com/");
+		//waitForPageLoad(10);
+
+	}
+
```
Comment (snelloru) [inline SeleniumPactice/April2022SeleniumSession/src/main/java/seleniumsessions/CustomizedWait.java]: Can we remove these empty lines here?

**Review by snelloru — CHANGES_REQUESTED** [review]
Please address these comments

File: SeleniumPactice/April2022SeleniumSession/src/main/java/seleniumsessions/CustomizedWait.java (hunk 1 above)
Comment (VinithaDar) [reply SeleniumPactice/April2022SeleniumSession/src/main/java/seleniumsessions/CustomizedWait.java:6]: yes, we need for webdrivermanager method.

**Comment by VinithaDar** [comment]
@snelloru I addressed all your comments.

**Review by snelloru — APPROVED** [review]
Looks good to me

**Pull request merged**